.drone.yml

```diff
@@ -21,8 +22,10 @@ pipeline:
   test:
     image: golang:1
     environment:
+      - GO111MODULE=on
       - CGO_ENABLED=0
     commands:
+      - go mod download
       - go vet
       - go test -cover
```

Comment:
Not part of this PR, but in recent versions of Go `go test` and `go vet` can be combined with `go test -vet all`.